### ISSUE TITLE
Panic during apply

### ISSUE THREAD

**Issue by macbutch** [diggerhq/digger#1146]
Thanks @motatoes, for your help with my previous issue. It looks like plans are succeeding now but when I apply I see this:

```
Lock provider has been created successfully
Using GitHub.
GitHub context parsed successfully
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x14e27b7]
goroutine 1 [running]:
github.com/diggerhq/digger/cli/pkg/digger.UpdateStatusComment({0xc000a50c00?, 0xe, 0x36000a0?}, 0x0?, {0x2594950, 0xc000a4e510}, 0xffffffffffffffff?)
	/home/runner/work/digger/digger/cli/pkg/digger/digger.go:174 +0x117
main.gitHubCI({0x2587848, 0xc00012a420}, {0x2586e58, 0xc000b76990}, {0x2586e88, 0xc000b7ecc0}, {0x25745c0?, 0xc000b86000})
	/home/runner/work/digger/digger/cli/cmd/digger/main.go:135 +0xe31
main.glob..func2(0xc000159e00?, {0x1f47b06?, 0x4?, 0x1f47b0a?})
	/home/runner/work/digger/digger/cli/cmd/digger/default.go:23 +0x205
github.com/spf13/cobra.(*Command).execute(0x35d2980, {0x363aca0, 0x0, 0x0})
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:987 +0xaa3
github.com/spf13/cobra.(*Command).ExecuteC(0x35d3220)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1115 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/runner/go/pkg/mod/github.com/spf13/cobra@v1.8.0/command.go:1039
main.main()
	/home/runner/work/digger/digger/cli/cmd/digger/main.go:819 +0xf4
```

This was triggered by a `digger apply` comment after a successful plan on v0.3.26.

**Comment by motatoes** [comment]
Hey @macbutch sorry for that! Any chance you can share what your digger.yml and workflow file looks like ?

**Comment by motatoes** [comment]
Hey @macbutch appologies for all this null pointer madness, we have been revamping our commentOps flow and introduced some big new features and in the process introduced a bunch of edgecases unacconted for. This last one was due to a nil value we were not handling in workflow url. It should have bene resolved in #1148 

If you are using digger cloud triggering another plan/apply should work for you. But just in case try to update to latest version and hopefully it will resolve the issue [v0.3.27](https://github.com/diggerhq/digger/releases/tag/v0.3.27)

Please reopen this if still persists! 🙏

**Comment by macbutch** [comment]
Thanks @motatoes! I can see you guys are making a lot of progress - all good on my end. Thanks for the fix!
